scripts/deploy.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+const { spawn } = require("node:child_process");
+const os = require("os")
+const utils = require("./utils")
+const Witnet = require("witnet-toolkit")
+
+const addresses = require("../witnet/addresses.json")
+const assets = require("../witnet/assets")
+const requests = Witnet.Dictionary(Witnet.Artifacts.Class, assets?.requests)
+
+function cmd(...command) {
+    let p = spawn(command[0], command.slice(1));
+    return new Promise((resolve) => {
+        p.stdout.on("data", (x) => {
+            process.stdout.write(x.toString());
+        });
+        p.stderr.on("data", (x) => {
+            process.stderr.write(x.toString());
```

Comment:
Should we reject if there's data on `stderr`?
```suggestion
            process.stderr.write(x.toString());
            reject()
```